packages/editor/src/components/collab-sidebar/comment-form.js

```diff
@@ -50,6 +51,20 @@ function CommentForm( {
 		<VStack
 			className="editor-collab-sidebar-panel__comment-form"
 			spacing="4"
+			as="form"
+			onSubmit={ ( event ) => {
+				event.preventDefault();
+				onSubmit( inputComment );
+				setInputComment( '' );
+			} }
+			onKeyDown={ ( event ) => {
```

Comment:
Can we move this event handler to `TextareaAutosize`? For example, even when the cancel button or replay button is focused, the shortcut is executed, causing an error.


https://github.com/user-attachments/assets/24eacf4e-1037-4245-8d90-6957afefdd35